spec/models/hold_spec.rb

```diff
@@ -71,4 +71,11 @@
     hold.record['fields']['status'] = 'ACTIVE'
     expect(hold.status_code).to eq 'ACTIVE'
   end
+
+  it 'has a bib summary' do
+    hold.record['fields']['bib']['fields']['title'] = 'A wonderful title'
+    hold.record['fields']['item']['fields']['call']['fields']['dispCallNumber'] = 'ABC123'
+
+    expect(hold.bib_summary).to eq 'A wonderful title (ABC123)'
```

Comment:
@cdmo wondering if there are items with no callnumber?

Reply:
we have items in our catalog without a call number? I can change if that's the case.